cypress/support/index.ts

```diff
@@ -123,7 +115,6 @@ declare global {
        */
       createFakeLibrarySession(): void;
       createFakeAuthenticatedSession(): void;
-      createFakeLibrarySession(): void;
```

Comment:
It seems the dublication was introduced due to an error while rebasing, according to the original PR